configurations/netvm/default.nix

```diff
@@ -0,0 +1,82 @@
+{
+  nixpkgs,
+  microvm,
+  system,
+}:
+nixpkgs.lib.nixosSystem {
+  inherit system;
+  modules = [
+    # TODO: Enable only for development builds
+    ../../modules/development/authentication.nix
+    ../../modules/development/ssh.nix
+    ../../modules/development/packages.nix
+
+    microvm.nixosModules.microvm
+
+    ({pkgs, ...}: {
+      networking.hostName = "netvm";
+      system.stateVersion = "22.11";
+
+      # For WLAN firmwares
+      hardware.enableRedistributableFirmware = true;
+
+      microvm.hypervisor = "crosvm";
+
+      networking.enableIPv6 = false;
+      networking.interfaces.eth0.useDHCP = true;
+      networking.firewall.allowedTCPPorts = [22];
+
+      #users.users."graf" = {
+      #    isNormalUser = true;
+      #    password = "ghaf";
+      #    extraGroups = ["wheel"];
+      #};
+
+      # TODO: Idea. Maybe use udev rules for connecting
+      # USB-devices to crosvm
+
+      # TODO: Move these to target-specific modules
+      # microvm.devices = [
+      #   {
+      #     bus = "usb";
+      #     path = "vendorid=0x050d,productid=0x2103";
+      #   }
+      # ];
+      # microvm.devices = [
+      #   {
+      #     bus = "pci";
+      #     path = "0001:00:00.0";
+      #   }
+      #   {
+      #     bus = "pci";
+      #     path = "0001:01:00.0";
+      #   }
+      # ];
+
+      # x86 Laptop
```

Comment:
please mark this with `# TODO: move to user specified module - depending on the user x86 laptop pci path`